spyder/widgets/sourcecode/codeeditor.py

```diff
@@ -1925,6 +1925,11 @@ def fix_indent(self, forward=True, comment_or_string=False):
             cursor.movePosition(QTextCursor.PreviousBlock)
             prevtext = to_text_string(cursor.block().text()).rstrip()
 
+            # Remove inline comment
+            inline_comment = prevtext.find('#')

```

Comment:
Is this enough to detect an inline comment? What if you have a line like

    # foo

?